net2/HostManager.js

```diff
@@ -1408,6 +1428,27 @@ module.exports = class HostManager {
                   }
                 }
               }
+
```

Comment:
VPN Connect alarm should be generated only if the connection is successfully reestablished.